calcom/cal.com
# feat: checkbox to count managed events for team-wide limits

## What does this PR do?

Adds a checkbox to team-wide bookings limits so admins can choose if bookings from managed event types are counted for the time-wide booking limits. 

![Screenshot 2024-10-03 at 11 48 58 AM](https://github.com/user-attachments/assets/d5c20d9f-d94b-4e9c-890e-18c5bb50ba81)

Also fixes the following bug:
<img width="460" alt="Screenshot 2024-10-04 at 8 54 32 PM" src="https://github.com/user-attachments/assets/449e4144-515a-4f98-b48a-f48d0a057ee9">
How to reproduce bug? 
- Set schedule timezone to Singapore  (any + timezone)
- Add event type limit, 1 per year 
- book one meeting
- load availability again and see that it's still shows slots for this year, when trying to confirm the booking you get an error

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have added a Docs issue: https://github.com/calcom/docs/issues/139
